Assets/MixedRealityToolkit-Examples/Demos/StandardShader/Scripts/Editor/MixedRealityToolkit.Examples.Demos.StandardShader.Inspectors.asmdef

```diff
@@ -0,0 +1,12 @@
+{
+    "name": "MixedRealityToolkit.Examples.Demos.StandardShader.Inspectors",
```

Comment:
Please update this to match our other asmdef names (they start with `Microsoft.MixedReality.Toolkit` instead of `MixedRealityToolkit`). Probably `Microsoft.MixedReality.Toolkit.Examples.Demos.StandardShader.Inspectors` (though that's a little long haha).

Reply:
Haha, sounds perfect to me. I'll fix that!